### PR TITLE
FIX: Desk Default content template is not available in planning scheduled export. [SDCP-495]

### PR DESCRIPTION
**Pull request by GyanP**

### CODE REVIEW

**Review by petrjasek — APPROVED** [review]
test would be great

**Comment by GyanP** [comment]
> test would be great

sure

File: features/desks.feature [hunk 1]
```diff
@@ -754,3 +754,35 @@ Feature: Desks
                 ]
             }
         """
+    @wip
```
Comment (petrjasek) [inline features/desks.feature:757]: :eyes:

**Review by petrjasek — APPROVED** [review]

File: features/desks.feature (hunk 1 above)
Comment (GyanP) [reply features/desks.feature]: just removed.

**Pull request merged**